README.md

```diff
@@ -293,14 +281,14 @@ Now quorum is lost. The etcd operator will start to recover the cluster by:
 ```
 $ kubectl get pods
 NAME                             READY     STATUS     RESTARTS   AGE
-etcd-cluster-0005                0/1       Init:0/2   0          11s
+etcd-cluster-0002                0/1       Init:0/2   0          11s
```

Comment:
No walkthrough. This will be a maintenance burden for us. Adding sections, moving sections and deleting sections will affect other sections. Also I think most users just want to paste the commands and go over the doc quickly. No one really cares about if the name of the pod is 1 or 2. Also this one should be 002 anyway since we recreate a cluster with backup enabled in this section.